aggregator/src/param.rs

```diff
@@ -0,0 +1,58 @@
+use snark_verifier::loader::halo2::halo2_ecc::fields::fp::FpStrategy;
+
+pub(crate) const LIMBS: usize = 3;
+pub(crate) const BITS: usize = 88;
+
+#[derive(serde::Serialize, serde::Deserialize, Clone, Debug)]
+/// Parameters for aggregation circuit and compression circuit configs.
+pub struct ConfigParams {
+    pub strategy: FpStrategy,
+    pub degree: u32,
+    pub num_advice: Vec<usize>,
+    pub num_lookup_advice: Vec<usize>,
+    pub num_fixed: usize,
+    pub lookup_bits: usize,
+    pub limb_bits: usize,
+    pub num_limbs: usize,
+}
+
+impl ConfigParams {
+    pub(crate) fn _aggregation_param() -> Self {
+        Self {
+            strategy: FpStrategy::Simple,
+            degree: 23,
+            num_advice: vec![8],
+            num_lookup_advice: vec![1],
+            num_fixed: 1,
+            lookup_bits: 20,
+            limb_bits: 88,
+            num_limbs: 3,
+        }
+    }
+
+    pub(crate) fn compress_wide_param() -> Self {
```

Comment:
are these functions used by only testing or the zkevm super circuit proving?

Reply:
https://github.com/scroll-tech/zkevm-circuits/blob/2f288d5c8a365d415107f6358ec5f9130907f929/aggregator/src/compression/circuit.rs#L84

this one is used as the default config if the config file fail to deserialize.
The other two are not used.

(in general it would be better if we can pass a param to `configure()`  so that it can load different parameters...